snapshot/lib/snapshot/test_command_generator_base.rb

```diff
@@ -80,6 +81,17 @@ def derived_data_path
         Snapshot.cache[:derived_data_path] ||= (Snapshot.config[:derived_data_path] || Dir.mktmpdir("snapshot_derived"))
       end
 
+      def result_bundle_path
+        unless Snapshot.cache[:result_bundle_path]
+          path = File.join(Snapshot.config[:output_directory], Snapshot.config[:scheme]) + ".test_result"
```

Comment:
Can we include the current language/locale in the path? The snapshot test scheme will be executed per language and we would like to save each result_bundle, rather than having them overwritten.

Reply:
We should also include the device as well, as multiple devices can be specified to snapshot.

Reply:
@sirnacnud I can do the current language/locale part but I don't think I can do the devices part 😬 The new xcodebuild commands take multiple devices and one result bundle path so I'm not able to make a separate path for each device. So I'm assuming that there is only one result bundle path since there is only one xcodebuild. Thoughts?